docs/platform-integration/essentials-getting-started.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
I'd delete this doc. It's no longer required. There's no overall setup required to use this functionality in MAUI.